structured/sql_api.proto

```diff
@@ -93,6 +93,6 @@ message SQLRequest {
 
 message SQLResponse {
   optional SQLResponseHeader header = 1 [(gogoproto.nullable) = false, (gogoproto.embed) = true];
-  // The result set.
+  // The result set for the last Cmd in the request.
```

Comment:
Each result set definitely needs the list of columns returned. I totally missed that the list of columns returned was not part of the response.